docs/extend/extend-apiml/api-mediation-onboard-a-spring-boot-rest-api-service.md

```diff
@@ -388,7 +389,21 @@ As an API service developer, you set multiple configuration settings in your app
       * `metadata-map.routed-services.<prefix>.serviceUrl`
 
           Both gateway-url and service-url parameters specify how the API service endpoints are mapped to the API gateway endpoints. The service-url parameter points to the target endpoint on the gateway.
-
+      * `eureka.instance.metadata-map.apiml.apiInfo.apiId`
+      
+          Specifies the API identifier that is registered in the API Mediation Layer installation. The API ID uniquely identifies the API in the API Mediation Layer.
+          The same API can be provided by multiple service. The API ID can be used to locate the same APIs that are provided by different services. The creator of the API defines this ID.
```

Comment:
Please make sure that the parameter description for all enablers is consistent. ' The same API can be provided by multiple service.' should be changed to ' The same API can be provided by multiple serviceS.' There is probably more occurrences.

Reply:
Fixed the "s" for all Docs.